packages/common/src/utils/filters.ts

```diff
@@ -67,8 +70,8 @@ export const hasNestedGroups = (filters: Filters): boolean => {
         return items.some(isFilterGroup);
     };
     return (
-        (!!filters.dimensions && hasGroups(filters.dimensions)) ||
-        (!!filters.metrics && hasGroups(filters.metrics))
+        (!!filters.metrics && hasGroups(filters.metrics)) ||
```

Comment:
why have we removed the `(!!filters.dimensions && hasGroups(filters.dimensions))` result ?

Reply:
good catch, that's a mistake when resolving the conflict.